model/pipeline/src/main/proto/metrics.proto

```diff
@@ -118,6 +118,13 @@ message MonitoringInfoSpecs {
         value: "The total estimated execution time of the ptransform"
       } ]
     }];
+
+    // TODO(ajamato): Add the PTRANSFORM name as a required label after
+    // upgrading the python SDK.
+    USER_DISTRIBUTION_COUNTER = 6 [(monitoring_info_spec) = {
+      urn: "beam:metric:user_distribution:",
+      type_urn: "beam:metrics:distribution_int_64",
```

Comment:
nit: `int64` is standard (versus `int_64`)

Reply:
That's a good catch. Though I'd prefer not to do this change within this PR to avoid changes irrelevant to original intent.